CONTRIBUTING.md

```diff
@@ -13,7 +13,11 @@ section.
 ## Modules
 
 * Prefer the use of the singular form in module name (or use "multi")
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
+* In the `__openerp__.py`  manifest file:
+  * Avoid empty keys
+  * Make sure it has the `license` key
+  * Make sure the `author` key has appended the text `,Odoo Community Association (OCA)`
```

Comment:
Neither am I. I'm using `appended` as a verb, not an adjective. I think it's clearer to use a verb. Maybe it should be rephrased?:

    * Make sure the text `,Odoo Community Association (OCA)` is appended to the `author` text.